main.tf

```diff
@@ -68,6 +71,7 @@ resource "yandex_compute_instance" "vps" {
   metadata = {
     ssh-keys = "${var.ssh_username}:${file("${var.ssh_pubkey}")}"
```

Comment:
why need ssh-keys metadata if the user is created via cloud-init ?

Reply:
I saved two variants because some people may want to use only add ssh-key to exist user(without using cloud-init file), or create new by cloud-init config file.

Reply:
ok